tests/bricks/test_bricks.py

```diff
@@ -469,3 +469,22 @@ def test_linear_nan_allocation():
     b2 = linear.params[1].get_value()
     numpy.testing.assert_equal(w1, w2)
     numpy.testing.assert_equal(b1, b2)
+
+
+def test_merge():
```

Comment:
That can be just as well made a doctest. All other parallel brick now have  a doctest, so I think we should stick to this tradition.